configuration/observability.mdx

```diff
@@ -81,3 +81,40 @@ Currently, we support the following tracing backends:
 Enable tracing via the values described in the [Tracing configuration](/configuration/overview#tracing) and point Flipt to your configured collector to record spans.
 
 There are [examples](https://github.com/flipt-io/flipt/tree/main/examples/tracing) provided in the main GitHub repository showing how to set up Flipt with each of the supported tracing backends.
+
+## Audit Events
+
+Starting from `v1.21.0`, Flipt supports sending audit events to configured sinks. Audit events have the following structure:
+
+```json
+{
+  "version": "v0.1",
+  "metadata": {
+    "type": "flag",
+    "action": "created",
+    "actor": {
+      "ip": "127.0.0.1",
+      "method": "none",
+      "email": "joe@example.com"
+    }
+  },
+  "payload": {
+    "description": "audit events feature",
+    "enabled": true,
+    "key": "my-audit-event",
+    "name": "myauditevent",
+    "namespace_key": "default"
+  }
+}
+```
+
+- `version` : describes the version of the audit event structure. We do not expect too many changes to the structure of the audit event
+- `metadata`: information concerning the `type` and `action` taken with some identity metadata in the `actor` object
+- `payload` : the actual payload used to interact with the `Flipt` server for certain auditable events
+
+
+Currently, we support the following sinks for audit events:
+
+- Log File: the audit events are JSON encoded and new-line delimited
```

Comment:
what do you think about linking to the section in the configuration/overview docs that describe how to configure the log file sink?

Reply:
@markphelps Which one are you referring to? Is this within the main repo?

Reply:
i meant just a relative link to https://github.com/flipt-io/flipt-docs/pull/44/files#diff-5f9f59024a18f654eab5f4c7e1e55301570a261c9bc66af19b00836aada4ab6aR83

I think the url will be `/docs/configuration#audit-events` or similar.

You can run `mintlify dev` from this repo (after installing mintlify via the README) to see how everything will render/link up

Reply:
Done! Thanks Mark.